data_structures/src/chain.rs

```diff
@@ -2726,6 +2726,35 @@ pub struct ChainState {
     pub superblock_state: SuperBlockState,
 }
 
+impl ChainState {
+    /// Return the number of consolidated blocks
+    pub fn block_number(&self) -> u32 {
+        u32::try_from(self.block_chain.len()).unwrap()
+    }
+
+    pub fn get_chain_beacon(&self) -> CheckpointBeacon {
```

Comment:
Missing doc

Reply:
The code should not compile with missing docs, opened PR #1491 to fix that.